### ISSUE TITLE
cant download from vbox7

### ISSUE THREAD

**Issue by waitforminute** [ytdl-org/youtube-dl#26063]
- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2020.06.16.1**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[generic] article:3004067e04: Requesting header
[generic] article:3004067e04: Downloading webpage
[generic] article:3004067e04: Extracting information
[Instagram] CCVnyTuJqB8: Downloading webpage
```

## Description

I can't download this video, for example https://www.vbox7.com/play:317664aff4

**Comment by thansk** [comment]
I also tested this and it happens with all videos on this website.
I get this, using the -F option:
```
[Vbox7] xxx: Downloading JSON metadata
[Vbox7] xxx: Downloading webpage
WARNING: unable to extract JSON-LD ; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
WARNING: unable to extract thumbnail URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
[info] Available formats for xxx:
format code  extension  resolution note
0            mpd        unknown    
```
running 2020.06.16.1

**Comment by dirkf** [comment]
With PR #32710:
```console
$ python -m youtube_dl -vF 'https://www.vbox7.com/play:317664aff4'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-vF', u'https://www.vbox7.com/play:317664aff4']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 630da9eb7
[debug] Python 2.7.18 (CPython i686 32bit) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial - OpenSSL 1.1.1w  11 Sep 2023 - glibc 2.15
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[Vbox7] 317664aff4: Downloading JSON metadata
[Vbox7] 317664aff4: Downloading MPD manifest
[Vbox7] 317664aff4: Downloading webpage
[info] Available formats for 317664aff4:
format code  extension  resolution note
dash-5       m4a        audio only [eng] DASH audio  131k , m4a_dash container, mp4a.40.2
dash-4       mp4        256x144    [eng] DASH video  130k , mp4_dash container, avc3.4D400C, 25fps, video only
dash-3       mp4        426x240    [eng] DASH video  398k , mp4_dash container, avc3.4D4015, 25fps, video only
dash-2       mp4        852x480    [eng] DASH video  796k , mp4_dash container, avc3.4D401E, 25fps, video only
dash-1       mp4        1280x720   [eng] DASH video 1785k , mp4_dash container, avc3.4D401F, 25fps, video only
144p         mp4        144p       
240p         mp4        240p       
480p         mp4        480p       
720p         mp4        720p       (best)
$
```
